### PR TITLE
Fix some PMD, checkstyle and remove some redundant code 

### PR DESCRIPTION
**Pull request by ghyl**

### CODE REVIEW

File: src/test/java/nl/tudelft/ti2206/semgr8/model/highscore/HighscoreListTest.java [hunk 1]
```diff
@@ -1,23 +1,21 @@
...
+import org.junit.Test;
 
 import java.io.File;
 import java.io.FileNotFoundException;
 import java.util.ArrayList;
 import java.util.Scanner;
 
-import org.junit.Before;
-import org.junit.Test;
+import static org.junit.Assert.*;
```
Comment (LeonWubben) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/highscore/HighscoreListTest.java]: '.*' for import.
Looking at checkstyle on this branch the same problem is in HighscoreTest and AnimationHandler. Don't know if these are already fixed in other prs.
To deactivate this in IntelliJ. Settings -> Other settings -> Save actions -> Uncheck "Organize imports"

**Comment by LeonWubben** [comment]
Looks good, nothing much of interesting happening. 
Can be merged after the import style is fixed.

File: src/test/java/nl/tudelft/ti2206/semgr8/model/highscore/HighscoreListTest.java (hunk 1 above)
Comment (ghyl) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/highscore/HighscoreListTest.java]: wasn't sure too. but I'll just fix it in this branch now

**Comment by ghyl** [comment]
No more checkstyleviolations! will merge when travis is done.

**Pull request merged**